API.md

```diff
@@ -8140,6 +8141,22 @@ public readonly distribution: IDistribution;
 
 ---
 
+##### `additionalMetricsEnabled`<sup>Optional</sup> <a name="additionalMetricsEnabled" id="cdk-monitoring-constructs.CloudFrontDistributionMetricFactoryProps.property.additionalMetricsEnabled"></a>
+
+```typescript
+public readonly additionalMetricsEnabled: boolean;
```

Comment:
Should we make this naming consistent with the existing "advanced metrics" in API Gateway?

Reply:
The [docs](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/viewing-cloudfront-metrics.html#monitoring-console.distributions-additional) refer to them as "additional metrics", so I stuck with that.